### DISCUSSION TITLE
No pronouns

### DISCUSSION THREAD

**Discussion by gingershaped** [community/community#51883] [Mobile]
Pronouns are not shown in the Profile tab.

![Screenshot_20230403-153214.png](https://user-images.githubusercontent.com/75683114/229608849-12116ecc-0f18-4c24-ba5f-a3de6beb121d.png)

**Comment by iskae** [comment]
Hello @GingerIndustries 👋 

Thanks so much for reaching out ❤️ 

We already have an internal tracking issue for this and we will try to bring it to mobile as soon as possible 👍 

Please stay tuned for the future releases 😊 

Cheers!

**Reply by gingershaped** [reply]
Cool, thanks!
